sass/_aesop.scss

```diff
@@ -1,375 +1,392 @@
         }
-	}
-    
-    &.aesop-component-chat {
-        margin-bottom: 0;
-        margin-top: 0;
-        
-        + .aesop-component-chat {
-            margin-top: -10px;
+      }
+
+      blockquote {
+        opacity: 0;
+        transition: opacity 200ms;
+        @media screen and (max-width: $width-tablet) {
+          transform: none !important;
         }
+      }
     }
+  }
+
+  &.aesop-content-component {
+    * {
+      color: inherit;
+    }
+
+    .aesop-content-comp-inner {
+      padding: 30px 0;
+      width: 90%;
+
+      &.aesop-content {
+        @include site-width;
+      }
+    }
+  }
+
+  &.aesop-component-chat {
+    margin-bottom: 0;
+    margin-top: 0;
+
+    + .aesop-component-chat {
+      margin-top: -10px;
+    }
+  }
 }
 
 .aesop-component-chat-container {
-    width: 90%;
+  width: 90%;
 }
 
 .aesop-stacked {
-    &-img {
-        background-attachment: fixed;
-        background-position: 50%;
-        background-repeat: no-repeat;
-        background-size: cover;
-        height: 500px; // fallback for browsers that don't support vh
-        height: 100vh;
-        left: 0;
-        overflow: hidden;
-        position: relative;
-        top: 0;
-        width: 100%;
+  &-img {
+    background-attachment: fixed;
+    background-position: 50%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 500px; // fallback for browsers that don't support vh
+    height: 100vh;
+    left: 0;
+    overflow: hidden;
+    position: relative;
+    top: 0;
+    width: 100%;
 
-        @media screen and (max-width: $width-tablet) {
-            background-attachment: scroll;
-            background-size: contain !important; // overrides inline style set by js
-        }
-    }
-    
-    &-caption {
-        background-image: linear-gradient(rgba(0, 0, 0, .5), rgba(0, 0, 0, 0));
-        color: #fff;
-        left: 0;
-        padding: 20px 0 80px;
-        position: absolute;
-        right: 0;
-        text-align: center;
+    @media screen and (max-width: $width-tablet) {
+      background-attachment: scroll;
+      background-size: contain !important; // overrides inline style set by js
     }
+  }
+
+  &-caption {
+    background-image: linear-gradient(rgba(0, 0, 0, .5), rgba(0, 0, 0, 0));
+    color: #fff;
+    left: 0;
+    padding: 20px 0 80px;
+    position: absolute;
+    right: 0;
+    text-align: center;
+  }
 }
 
 .aesop-img-enlarge {
-    display: none; // do something with this :/
+  display: none; // do something with this :/
 }
 
 .aesop-photoset-gallery-wrap .aesop-photoset-caption .aesop-photoset-caption-title,
 .aesop-photoset-gallery-wrap .aesop-photoset-caption .aesop-photoset-caption-caption {
-    // overrides 
+  // overrides
 }
 
 .aesop-parallax-sc-caption-wrap {
-    background: rgba(0, 0, 0, .8);
-    color: #fff;
-    font-family: $font-stack-content;
-    @include rem(font-size, $font-size-normal);
-    padding: 10px 20px;
+  background: rgba(0, 0, 0, .8);
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets